ke/compiler.py

```diff
@@ -259,14 +254,21 @@ def compile_operator(o, macros):
         else:
             min = int(min)
             max = int(max)
-        return asm.Multiple(min, max, True, sub)
+        
+        return asm.Multiple(min, max, get_greediness_by_name_of_operator(o.name), sub)
     if o.op_name not in builtin_operators:
         raise CompileError("Operator %s does not exist" % o.op_name)
     return builtin_operators[o.op_name](o.name, sub)
 
 
+def get_greediness_by_name_of_operator(name):
+    #TODO Complete rest of the operators
+    return name != 'fewest' and name != 'f'
+
 def compile_macro(macro, macros):
-    if macro.name not in macros:
+    if "#repeat:" in macro.name or "#r:" in macro.name:
```

Comment:
make sure to detect this as an error: `[c #d][#r:1:2]`

Reply:
Not sure if you're just wanting a test or it's a bug, but tests passes with:
`with pytest.raises(re.error):
        assert ke.re("c #d][#r:1:2]")`